conda.bzl

```diff
@@ -26,6 +26,21 @@ CONDA_INSTALLER_FLAGS = {
     "Linux": ["-b", "-f", "-p", "{}"],
 }
 
+MINIFORGE_MAJOR = "3"
+MINIFORGE_MINOR = "4.10.3-7"
+MINIFORGE_SHA = {
+    # TODO(jiawen - DONOTMERGE): These don't seem to be verified. Add paths for other OSs and archs before submitting.
+    "MacOSX": {
+        "x86_64": "786de9721f43e2c7d2803144c635f5f6e4823483536dc141ccd82dbb927cd508",
```

Comment:
These are nonsense values copy-pasted from above. They are passed to `rctx.download` but it doesn't seem to be checked?

Reply:
According to [docs](https://docs.bazel.build/versions/main/skylark/lib/repository_ctx.html#download) the `sha256` passed to `rctx.download` must match the one of the actual file, so I assume the download action should check the integrity itself. If that's not the case then it's kinda weird.

Reply:
🤷 . In any case, I updated the hashes.